.eslintrc

```diff
@@ -5,11 +5,9 @@
   "extends": [
     "eslint:recommended",
     "plugin:import/typescript",
-    "plugin:react-app/recommended",
     "plugin:@typescript-eslint/eslint-recommended",
     "plugin:@typescript-eslint/recommended",
-    "prettier",
-    "prettier/@typescript-eslint"
```

Comment:
config for prettier/@typescript-eslint er slått sammen med prettier https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21